packages/core/cms/faststore/content-types.json

```diff
@@ -110,8 +110,8 @@
                         }
                       }
                     },
-                    "deliveryCustomLabels": {
-                      "title": "Filter options labels",
+                    "deliveryMethods": {
```

Comment:
I've made this change because it isn't only for labels anymore and it's related to delivery _methods_